test/MANACrowdsale.js

```diff
@@ -80,13 +83,13 @@ contract('MANACrowdsale', function ([_, wallet, wallet2, buyer, purchaser, buyer
 
     await crowdsale.buyTokens(buyer, {value, from: purchaser})
     balance = await token.balanceOf(buyer)
-    balance.should.be.bignumber.equal(value.mul(rate))
+    balance.should.be.bignumber.equal(value.mul(initialRate))
 
     await advanceToBlock(startBlock + 4)
 
     await crowdsale.buyTokens(buyer2, {value, from: purchaser2})
     balance = await token.balanceOf(buyer2)
-    const rateAtBlock5 = rate.minus(rateStepDecrease.mul(5))
+
```

Comment:
Could you add another test where blockRange is **greater** than rateRange? This should cause the test to fail without the change pointed above .